src/helpers/transformGeoInfo.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * geoData - {object}
+ * ip: string
+ * isp: string
+ * location - {object Location}
+ * * city: string
+ * * country: string
+ * * geonameId: number
+ * * lat: number
+ * * lng: number
+ * * postalCode: string
+ * * region: string
+ * * timezone: string
+ */
```

Comment:
It was an example. You should remove this comment or do it proper way https://jsdoc.app/